LCARS/Client/js/containers/BuildsPage/index.js

```diff
@@ -14,7 +14,7 @@ const Build = styled.div`
 
 const ProjectName = styled.span`
   display: inline-block;
-  width: 200px;
+  width: 300px;
```

Comment:
It's best to use `rem` rather than `px` as `rem` is scalable. To calculate it use `Target Size / Base Size = Value` where `Base Size` is equal to the `font-size` of the root element.

If the `font-size` is `10px` then calculating the above with would work like:
`300 / 10 = 30rem`

Reply:
This is the width of the span though?